ambry-server/src/integration-test/java/com.github.ambry.server/ServerSSLTest.java

```diff
@@ -120,6 +120,17 @@ public void endToEndSSLTest()
         clientSSLConfig1, clientSSLSocketFactory1, routerProps, testEncryption);
   }
 
+  /**
+   * Do endToEndTest with the last dataNode whose storeEnablePrefetch is true.
+   */
+  @Test
+  public void endToEndSSLTestWithPrefetch()
+      throws InterruptedException, IOException, InstantiationException, URISyntaxException, GeneralSecurityException {
+    DataNodeId dataNodeId = sslCluster.getClusterMap().getDataNodeIds().get(8);
```

Comment:
how come you don't do size - 1 here? Also I prefer that we have a function for this than rely on the knowledge of the code